lib/run/index.js

```diff
@@ -92,8 +110,10 @@ module.exports = function (options, callback) {
 
         // now start the run!
         runner.run(options.collection, {
-            stopOnFailure: options.bail, // LOL, you just got trolled ¯\_(ツ)_/¯
+            stopOnFailure: stopOnFailure, // LOL, you just got trolled ¯\_(ツ)_/¯
```

Comment:
Can be simplified as:
```
stopOnFailure: _.get(options, 'bail.failure', options.bail)
```

Reply:
Doesn't work, sets `stopOnFailure` to `{ folder: true }` in case of `--bail folder`